lib/evss/dependents/service.rb

```diff
@@ -3,34 +3,56 @@
 module EVSS
   module Dependents
     class Service < EVSS::Service
+      include Common::Client::Monitoring
       configuration EVSS::Dependents::Configuration
+      STATSD_KEY_PREFIX = 'api.evss.dependents'
 
       def retrieve
-        perform(:get, 'load/retrieve').body
+        with_monitoring do
+          perform(:get, 'load/retrieve')
+        end
+      rescue StandardError => e
+        handle_error(e)
```

Comment:
These will be handled by `EVSS::Service`s definition [here](https://github.com/department-of-veterans-affairs/vets-api/blob/dcb5b08b0df1bb6146092a923a4166b901c2b33a/lib/evss/service.rb#L30).

Is that what we want?

Reply:
i think so, how about making a helper method that does both `with_monitoring` and `handle_error` and put it in `EVSS::Service`?

Reply:
Yeah, I'll do that in a separate PR because it will involve me refactoring a good number of other services.